src/replace.js

```diff
@@ -57,12 +57,23 @@ function asIdentifier(path, filename) {
       return asTag(targetPath, filename)
     }
     case 'CallExpression': {
-      return asFunction(targetPath, filename)
+      const isCallee = targetPath.get('callee') === path
```

Comment:
`doSomething(codegen)` would cause the `CallExpression` branch to match even though it was the callee.